CMakeLists.txt

```diff
@@ -435,21 +449,37 @@ if(GPERF)
   find_package(Gperftools REQUIRED)
 endif()
 
-target_include_directories(devilution PUBLIC Source SourceS ${CMAKE_CURRENT_BINARY_DIR})
+target_compile_definitions(devilution PUBLIC SPDLOG_COMPILED_LIB)
+target_compile_definitions(${BIN_TARGET} PUBLIC SPDLOG_COMPILED_LIB)
+
+if(TRACE_LOGGING)

```

Comment:
Instead of doing this here, simply add `TRACE_LOGGING` to the `# Defines without value` list below